integration/Jenkinsfiles/utest_aurora.groovy

```diff
@@ -0,0 +1,88 @@
+/*
+ * (C) Copyright 2017 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     atimic
+ */
+
+properties([
```

Comment:
why the need of properties, compared to utest.groovy? fixes or improvements should be reported, behaviors should be maintained consistent together

here, it looks like an improvement, to be forwarded to utest.groovy and ftest.groovy as in sonar.groovy

Reply:
Same reason, just went with minimal pipeline done in [NXBT-2322](https://jira.nuxeo.com/browse/NXBT-2322) and as far as I know utest/ftest.groovy are not finished yet, theres' a PR on it #2056.
Aurora testing will be merged once tests will be ok and pipeline ready on our CI to do it (which includes general availability of files).